plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/task/MetalavaSignature.kt

```diff
@@ -43,7 +67,8 @@ internal object MetalavaSignature : MetalavaTaskContainer() {
                 doFirst {
                     val fullClasspath = (module.bootClasspath + module.compileClasspath).joinToString(File.pathSeparator)
 
-                    val sourcePaths = listOf("--source-path") + sources.joinToString(File.pathSeparator)
+                    val sourcePaths = listOf("--source-path") + inputs.files.joinToString(File.pathSeparator)
```

Comment:
Would we ever be accidentally be passing extra stuff to Metalava this way vs. passing `sourceFiles` directly?

Reply:
Sorry, this was an error. It should be `sourceFiles.joinToString(...)`